examples/Examples.ConfluentKafka/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Confluent.Kafka;
+using OpenTelemetry;
+using OpenTelemetry.Instrumentation.ConfluentKafka;
+using OpenTelemetry.Trace;
+
+ProducerConfig producerConfig = new();
+producerConfig.BootstrapServers = "localhost:9092";
+InstrumentedProducerBuilder<string, string> producerBuilder = new(producerConfig);
+
+ConsumerConfig consumerConfigA = new();
+consumerConfigA.BootstrapServers = "localhost:9092";
+consumerConfigA.GroupId = "group-a";
+consumerConfigA.AutoOffsetReset = AutoOffsetReset.Earliest;
+consumerConfigA.EnablePartitionEof = true;
+InstrumentedConsumerBuilder<string, string> consumerBuilderA = new(consumerConfigA);
+
+ConsumerConfig consumerConfigB = new();
+consumerConfigB.BootstrapServers = "localhost:9092";
+consumerConfigB.GroupId = "group-b";
+consumerConfigB.AutoOffsetReset = AutoOffsetReset.Earliest;
+consumerConfigB.EnablePartitionEof = true;
+InstrumentedConsumerBuilder<string, string> consumerBuilderB = new(consumerConfigB);
+
+string topic = $"test-topic-{Guid.NewGuid()}";
+
+// Configure exporter to export traces to Zipkin
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .AddConsoleExporter()
+    .AddZipkinExporter()
+    .AddKafkaProducerInstrumentation(producerBuilder)
+    .AddKafkaConsumerInstrumentation("consumer-a", consumerBuilderA)
+    .AddKafkaConsumerInstrumentation("consumer-b", consumerBuilderB)
```

Comment:
This is a bit cumbersome, as it requires 3 sequential initialization steps:

1. Initialize a consumer/producer builder.
2. Initialize the OTel SDK.
3. Initialize the consumer/producer.

Probably not a showstopper, but feels a bit weird to have that three-way DI dependency.